plugin/vcs/vcs.go

```diff
@@ -92,11 +92,27 @@ type RepositoryMember struct {
 	RoleProvider Type               `json:"roleProvider"`
 }
 
+// Repository is the API message for  repository info.
+type Repository struct {
+	ID       int64  `jsonapi:"primary,id"`
```

Comment:
As mentioned before, jsonapi should only appear under api module.

And here is the plugin module. API depends on plugin, not the other way around. Put it another way, plugin should have no knowledge of the API module.

If we reference jsonapi here, it essentially means we have a circular dependency, API -> plugin -> API.

A proper way to do this is to define a ExternalRepository struct under api/vcs.go